### PR TITLE
[DOC] Update transformer docstrings Boss

### PR DESCRIPTION
**Pull request by thayeylolu**
Docsprint Event

### CODE REVIEW

**Comment by fkiraly** [comment]
is this ready for review? Looks done

File: sktime/classification/dictionary_based/_muse.py [hunk 1]
```diff
@@ -46,46 +46,48 @@ class MUSE(BaseClassifier):
...
+    word_length:
+    norm_option:
+
     See Also
     --------
     WEASEL
 
+    References
+    ----------
+        .. [1] Patrick Schäfer and Ulf Leser, "Multivariate time series classification
```
Comment (mloning) [inline sktime/classification/dictionary_based/_muse.py:82]: I think there are too many spaces here
```suggestion
    .. [1] Patrick Schäfer and Ulf Leser, "Multivariate time series classification
```

**Review by mloning — COMMENTED** [review]
Hi @thayeylolu - looks all good to me, I'd like to get this into the next release, just have one minor comment.

**Review by mloning — APPROVED** [review]
It's not complete yet, but an improvement over the existing docstring, so can be merged as it I think!

**Review by fkiraly — APPROVED** [review]
Yes, why not. It will hopefully converge over time - let's not tick it off the list if it's not "complete complete".

**Comment by mloning** [comment]
Now merged @thayeylolu 🎉

**Pull request merged**